index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="https://cdn.jsdelivr.net/npm/phaser@3.15.1/dist/phaser-arcade-physics.min.js"></script>
+    <script src="main.js"></script>
```

Comment:
To Responsive:
<meta name="viewport" content="width=device-width, initial-scale=1.0">

Reply:
Tag meta with name="viewport" content="width=device-width, initial-scale=1.0